controllers/team_controller.py

```diff
@@ -11,6 +11,21 @@
 from renderers.team_renderer import TeamRenderer
 from template_engine import jinja2_engine
 
+import webapp2
+import random
+
+
+class RandomTeam(webapp2.RequestHandler):
+    """
+    Lovingly written by Caleb Denio (https://github.com/cjdenio)
```

Comment:
Generally, I wouldn't put this kind of thing in open source code.  Having attribution is fine, in fact it's a good idea, but that should come from a "contributors" file/page or the like, not from the individual source files.  🙂  If someone actually looking at the code needs to know who wrote it, `git blame` should suffice.

Reply:
I'll remove it; it's no problem :smile: